frontend/app/routes/_gcweb-app.personal-information._index.tsx

```diff
@@ -22,19 +24,19 @@ export const handle = {
 export async function loader({ request }: LoaderFunctionArgs) {
   const userId = await userService.getUserId();
   const userInfo = await userService.getUserInfo(userId);
-
+  const homeAddressInfo = await addressService.getAddressInfo(userId, userInfo?.homeAddress ?? '');
+  const mailingAddressInfo = await addressService.getAddressInfo(userId, userInfo?.mailingAddress ?? '');
   if (!userInfo) {
     throw new Response(null, { status: 404 });
   }
```

Comment:
Two comments:

- [ ] The retrieval of the home and mailing addresses should be moved below the `if (!userInfo)` check. We don't want to be checking for a home and mailing address if the `userInfo` does not exist.
- [ ] Rather than defaulting to an empty string if there is no home address or mailing address, perform a check to see if the `userInfo.homeAddress` and `userInfo.mailingAddress` exist first (similar to how it's currently done for `preferredLanguage`)

Reply:
Done